aidial_adapter_bedrock/llm/model/claude/v3/converters.py

```diff
@@ -32,87 +35,92 @@
     SystemMessage,
 )
 from aidial_adapter_bedrock.llm.tools.tools_config import ToolsMode
+from aidial_adapter_bedrock.utils.resource import Resource
 
 ClaudeFinishReason = Literal[
     "end_turn", "max_tokens", "stop_sequence", "tool_use"
 ]
 ImageMediaType = Literal["image/png", "image/jpeg", "image/gif", "image/webp"]
-IMAGE_MEDIA_TYPES: Set[ImageMediaType] = {
+IMAGE_MEDIA_TYPES: List[str] = [
```

Comment:
Why did you move to `List[str]` from `Set[ImageMediaType]`?

Reply:
The order of keys in the set isn't fixed from one program run to another. Since the set is printed in the user-facing error message, I need to remember to call `sorted` before printing the list of image types, so as to make sure that the error message is always the same. 

With a list I don't need to do it.

Besides, the lookup performance in short lists is comparable to the one in sets.